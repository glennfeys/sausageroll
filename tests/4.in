~ ~ ~ ~ ~ ~ ~ 
~ + +n+w+e+x~ 
~ + +s+ + +N~ 
~ ~ ~ ~ ~ ~ ~ 
