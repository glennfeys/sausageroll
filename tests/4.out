~ ~ ~ ~ ~ ~ ~ 
~ + +n+w+e+x~ 
~ + +s+ + +N~ 
~ ~ ~ ~ ~ ~ ~ 

~ ~ ~ ~ ~ ~ ~ 
~ +n+w+e+ + ~ 
~ +s+ + +x+W~ 
~ ~ ~ ~ ~ ~ ~ 

